### PR TITLE
Fixes #90 add developer terminal for macOS

### LINKED ISSUE #90: Implement developer terminal for macOS
currently "Open developer terminal" only works on Windows, we need to use `applescript` or something similar to do the same on macOS

### PR DESCRIPTION
**Pull request by anjannath**

### CODE REVIEW

**Comment by anjannath** [comment]
@praveenkumar can you please try this on macOS

**Comment by gbraad** [comment]
^^^ @praveenkumar I'd appreciate feedback

File: main.js [hunk 1]
```diff
@@ -388,6 +388,33 @@ prepareDevTerminalForPreset = async function(preset) {
...
+          shell: true,
+          stdio: 'ignore'
+        })
+
+        terminal.unref()
+      }
+    })
   } else {
     showNotification({
       body: "Only supported on Windows currently"
```
Comment (gbraad) [inline main.js]: Please update this line.

**Review by gbraad — CHANGES_REQUESTED** [review]
Contains a "WIndows only" remark

File: main.js (hunk 1 above)
Comment (anjannath) [reply main.js]: done

**Review by gbraad — APPROVED** [review]

**Pull request merged**